e2e-tests/content-strings/error-messages.js

```diff
@@ -375,6 +375,17 @@ export const ERROR_MESSAGES = {
           ABOVE_MAXIMUM: `The broker or company email cannot be more than ${MAXIMUM_CHARACTERS.EMAIL} characters`,
           INCORRECT_FORMAT: 'Enter the broker or company email address in the correct format, like name@example.com',
         },
+        [FIELD_IDS.INSURANCE.POLICY.BROKER_DETAILS.IS_BASED_IN_UK]: {
+          IS_EMPTY: 'Select if the broker you are using in the UK',
+        },
+        [FIELD_IDS.INSURANCE.POLICY.BROKER_DETAILS.POSTCODE]: {
+          IS_EMPTY: "Enter the broker's post code",
```

Comment:
this line says post code, line belows says postcode.  should it be standardised?

Reply:
sorry I mean the text for the error.  IS_EMPTY says post code and INCORRECT_FORMAT says postcode